raw/src/requests/restrict_chat_member.rs

```diff
@@ -0,0 +1,63 @@
+use types::*;
+use requests::*;
+
+/// Use this method to kick a user from a group or a supergroup.
+/// In the case of supergroups, the user will not be able to return to the group on
+/// their own using invite links, etc., unless unbanned first.
+/// The bot must be an administrator in the group for this to work.
+#[derive(Debug, Clone, PartialEq, PartialOrd, Serialize)]
+#[must_use = "requests do nothing unless sent"]
+pub struct RestrictChatMember {
+    chat_id: ChatRef,
+    user_id: UserId,
+    until_date: Option<i32>,
```

Comment:
Why is this `i32`?

Reply:
It's the unix time date, maybe we have to use a better format?